### ISSUE TITLE
Add affinity and tolerations to ChiPodTemplate

### ISSUE THREAD

**Issue by svmaris** [Altinity/clickhouse-operator#95]
Currently only Name, Containers and Volumes from the PodTemplate are copied to the Pod spec. I'm trying to use dedicated nodes for Clickhouse, so it would be nice if the affinity and tolerations are copied from the template, so I can do something like this:

```yaml
    podTemplates:
    - name: clickhouse-v19.6.2.11
      tolerations:
        - key: "dedicated"
          operator: "Equal"
          value: "clickhouse"
          effect: "NoSchedule"
      affinity:
        nodeAffinity:
          requiredDuringSchedulingIgnoredDuringExecution:
            nodeSelectorTerms:
            - matchExpressions:
              - key: dedicated
                operator: In
                values:
                - clickhouse
```

**Comment by sunsingerus** [comment]
This is an issue indeed, and it is addressed in next release to be made - [version 0.3](https://github.com/Altinity/clickhouse-operator/tree/release-0.3.0). `podTemplate` is switched to have `PodSpec` struct from `k8s.io/core/v1` as `podTemplate.spec`, so `affinity` and all the rest comes included.
[This example](https://github.com/Altinity/clickhouse-operator/blob/release-0.3.0/docs/examples/10-zones-aws-02-pod-per-host.yaml) is exactly about question discussed.

**Comment by alex-zaitsev** [comment]
0.3.0 is released, so templates now support full spec.
